xbmc/music/Song.cpp

```diff
@@ -91,12 +91,13 @@ void CSong::SetArtistCredits(const std::vector<std::string>& names, const std::v
   //Split the artist sort string to try and get sort names for individual artists
   std::vector<std::string> artistSort = StringUtils::Split(strArtistSort, CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_musicItemSeparator);
 
+    // Vector of possible separators in the order least likely to be part of artist name
+  const std::vector<std::string> separators{" feat. ", " ft. ", " Feat. ", " Ft. ",  ";", ":",
```

Comment:
Make it static maybe?
```suggestion
  static const std::vector<std::string> separators{" feat. ", " ft. ", " Feat. ", " Ft. ",  ";", ":",
```

Reply:
Fair.  Done, although the compiler will/should probably optimise it anyway.